application/src/main/java/org/togetherjava/tjbot/features/help/HelpSystemHelper.java

```diff
@@ -104,6 +107,15 @@ public HelpSystemHelper(Config config, Database database, ChatGptService chatGpt
         threadActivityTagNames = Arrays.stream(ThreadActivity.values())
             .map(ThreadActivity::getTagName)
             .collect(Collectors.toSet());
+
+        // tagsToIgnore list values are sanitized to avoid any config mismatch due to human error
+        List<String> tagsToIgnoreList = helpConfig.getTagsToIgnore()
+            .stream()
+            .map(String::toLowerCase)
+            .map(String::strip)
+            .toList();
+        tagsToIgnore = new HashSet<>(tagsToIgnoreList);
```

Comment:
y not only track `<category>*` tags, instead of configuring tags to ignore explicitly

Reply:
not sure i understand, we had a bunch of tags that weren't exactly added by user, so figured config would be one way to handle it without being explicit in code. Also gives flexibility in future to skip more tags if added.

Reply:
we have a list of category names from config so just track those